wlauto/external/uiauto/src/com/arm/wlauto/uiauto/Utility.java

```diff
@@ -0,0 +1,37 @@
+/*    Copyright 2013-2016 ARM Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+package com.arm.wlauto.uiauto;
+
+import java.util.logging.Logger;
+import android.os.Bundle;
+
+public final class Utility{
```

Comment:
"Utility" is too general a name. Maybe something like "UiAutoUtils" or something would be better.